rook/XMLDiff.py

```diff
@@ -18,9 +18,12 @@
 import sys
 import os
 import xml.etree.ElementTree as ET
+from Tester import Differ
 import DiffUtils as DU
+#CSWF Defined because otherwise lines of code get too long.
+CSWF = DU.compare_strings_with_floats
```

Comment:
lower case for "CSWF"? consistent with previous changes?

Reply:
changed to cswf